apps/vmq_server/src/vmq_config.erl

```diff
@@ -209,6 +209,8 @@ set_env(Node, App, Key, Val, true) when Node == node() ->
         case vmq_metadata:get(?DB, {Node, App, Key}) of
             undefined ->
                 #vmq_config{key = {Node, App, Key}, val = Val};
+            {error, no_matching_hook_found} ->
```

Comment:
Since we have removed metadata plugin(swc), we need to handle metadata related calls wherever it is made. These metadata related calls are mostly in config and retain modules which is not taken up as part of this effort. I just noticed few calls are also made due to vmq_reg_trie, I'll remove those calls in future commits as we need to get rid of vmq_reg_trie as we are now using vmq_reg_redis_trie.

vmq_reg_trie working is completely dependent on swc and clustering, so it is better to remove it as well.